contributing.md

```diff
@@ -29,6 +29,14 @@ For setups with local Docker environment you don't need Vagrant and VirtualBox.
 4. Visit [stream.local](https://stream.local) and login using `admin` / `password`.
 5. Activate the Stream plugin.
 
+#### Composer Timeout
+
+On the first setup of a fresh install, you may experience Composer 300s timeout when cloning WordPress Develop from source in order to cache. WordPress Develop repo is large and will consistently timeout when it is not cached, so it is recommended to temporarily remove the timeout for the first time installing:
+
+	export COMPOSER_PROCESS_TIMEOUT=0; npm install
```

Comment:
Could we remove the `export` prefix to ensure the timeout environment variable value applies only to this one specific command? That will also ensure users don't have to reset the global composer timeout value.